### ISSUE TITLE
Make undo work cmd-U (soft undo)

### ISSUE THREAD

**Issue by aronwoost** [aronwoost/sublime-expand-region#39]

**Comment by bizoo** [comment]
In ST3 (probably something similar is possible in ST2), you can use still use the begin_edit/end_edit.
But as these API methods are not documented, they could break in the next release.

This code force a new undo group, it was not tested extensively so maybe there are some issues with it.
And it works with multi-selection too :smile:
```python
    def run(self, edit):
        sels = ....
        token = edit.edit_token
        self.view.end_edit(edit)
        subedit = self.view.begin_edit(token, self.name())
        try:
            self.view.sel().clear()
            for sel in sels:
                self.view.sel().add(sel)
        finally:
            self.view.end_edit(subedit)
```

**Comment by aronwoost** [comment]
Closed with 99a05770db5823482987139bf937a61c6e4cfafb
